lib/main.coffee

```diff
@@ -14,7 +14,7 @@ module.exports =
       description: 'Remember the typed query when closing the fuzzy finder and use that as the starting query next time the fuzzy finder is opened.'
     useAlternateScoring:
       type: 'boolean'
-      default: false
+      default: true
       description: 'Use an alternative scoring approach which prefers run of consecutive characters, acronyms and start of words. (Experimental)'
```

Comment:
Then you may be interested to remove the word `experimental` next to autocomplete-plus Symbol provider. I think it's the default since big 1.0

https://github.com/atom/autocomplete-plus/blob/931841dfc3a3318cbb141f5c7abc5eee50aaeb28/lib/main.coffee#L101-L105